src/tools/dashboard/components/Dashboard.tsx

```diff
@@ -26,6 +30,41 @@ const EXPIRED_TIME = 1000 * 60 * 60 * 24 * 7; // 1 week
 
 const sortNewestFirst = (a: Job, b: Job) => b.timeCreated - a.timeCreated;
 
+const useDashboardEffect = () => {
```

Comment:
Wanted to pull that into the folder with all the other custom hooks but it's quite specific to here, and I'm also not sure how to name this. The issue pulling it too far is that the `useDashboardEffect();` call in the component render might not be obvious to what it does and keeping that just above or might be overlooked.
Open to change suggestions on that

Reply:
I think makes sense to keep it close to the rest of the dashboard code. Maybe moving to a new `dashboard/hooks/` directory like we similarly do with `useInitialFormParameters`?